src/environments/gridworlds.jl

```diff
@@ -0,0 +1,27 @@
+import ReinforcementLearningBase
+import .GridWorlds
+export RLBaseGridWorld
+
+mutable struct RLBaseGridWorld{T<:GridWorlds.AbstractGridWorld} <: ReinforcementLearningBase.AbstractEnv
+    env::T
+end
+
+Base.convert(::Type{ReinforcementLearningBase.AbstractEnv}, env::GridWorlds.AbstractGridWorld) = convert(RLBaseGridWorld, env)
+
+Base.convert(::Type{RLBaseGridWorld}, env::GridWorlds.AbstractGridWorld) = RLBaseGridWorld(env)
+
+ReinforcementLearningBase.get_state(env::RLBaseGridWorld) = GridWorlds.get_state(env.env)
+
+ReinforcementLearningBase.get_actions(env::RLBaseGridWorld) = GridWorlds.get_actions(env.env)
+
+ReinforcementLearningBase.get_reward(env::RLBaseGridWorld) = GridWorlds.get_reward(env)
+
+ReinforcementLearningBase.get_terminal(env::RLBaseGridWorld) = GridWorlds.get_terminal(env.env)
+
+ReinforcementLearningBase.get_legal_actions(env::RLBaseGridWorld) = GridWorlds.get_legal_actions(env.env)
+
+ReinforcementLearningBase.get_legal_actions_mask(env::RLBaseGridWorld) = GridWorlds.get_legal_actions_mask(env.env)
+
+ReinforcementLearningBase.reset!(env::RLBaseGridWorld) = GridWorlds.reset!(env.env)
+
+(env::RLBaseGridWorld)(action) = env.env(action)
```

Comment:
Use `env` instead of  `env` here. From the API level, RLBase can not be aware of the inner field of `env`.

Use `RLBase` here for consistency.

Reply:
>Use `env` instead of `env` here. From the API level, RLBase can not be aware of the inner field of env.

What do you mean by using `env` instead of `env`? Could you please explain this a little more?

Reply:
There is a bug in line 17. It should be `env.env` instead of `env`. Were you talking about that?

Reply:
Fixed bug on line 17 and changed `ReinforcementLearningBase` to `RLBase` for consistency.

Reply:
Now I see. Thanks.